packages/driver/src/cy/commands/request.js

```diff
@@ -277,7 +277,23 @@ module.exports = (Commands, Cypress, cy, state, config) => {
       // because we're handling timeouts ourselves
       cy.clearTimeout('http:request')
 
-      return Cypress.backend('http:request', requestOpts)
+      return Promise.try(() => {
+        // https://github.com/cypress-io/cypress/issues/6178
+        // Check if body is Blob.
+        // construct.name is added because the parent of the Blob is not the same Blob
+        // if it's generated from the test spec code.
+        if (requestOpts.body instanceof Blob || requestOpts?.body?.constructor.name === 'Blob') {
+          return Cypress.Blob.blobToBase64String(requestOpts.body).then((str) => {
```

Comment:
I first thought of writing code with `socket.io-parser` functions. But it seems to me that those functions are more of `private` functions than `public API`s. So, I decided to use the current code.